src/main/java/com/vaadin/demo/component/treegrid/TreeGridColumn.java

```diff
@@ -0,0 +1,58 @@
+package com.vaadin.demo.component.treegrid;
+
+import com.vaadin.demo.DemoExporter;
+import com.vaadin.demo.domain.DataService;
+import com.vaadin.demo.domain.Person;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.treegrid.TreeGrid;
+import com.vaadin.flow.router.Route;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Route("tree-grid-column")
+public class TreeGridColumn extends Div {
+
+    private List<Person> people = DataService.getPeople();
+
+    public TreeGridColumn() {
+        TreeGrid<Person> treeGrid = new TreeGrid<>();
+        treeGrid.setItems(getManagers(), this::getStaff);
+        treeGrid.addHierarchyColumn(Person::getFirstName);
+        treeGrid.addColumn(Person::getLastName);
+        treeGrid.addColumn(Person::getEmail);
```

Comment:
The headers are missing:
```suggestion
        treeGrid.addHierarchyColumn(Person::getFirstName).setHeader("First name");
        treeGrid.addColumn(Person::getLastName).setHeader("Last name");
        treeGrid.addColumn(Person::getEmail).setHeader("Email");
```